.github/workflows/bump-golang.yml

```diff
@@ -1,14 +1,12 @@
 ---
 name: Bump golang version
-
 on:
   workflow_dispatch:
   schedule:
     - cron: '0 20 * * 6'
 
 permissions:
-  pull-requests: write
-  contents: write
+  contents: read
```

Comment:
It uses a service account hence there is no need to leverage the permissions anymore